kafka/consumer/group.py

```diff
@@ -313,11 +313,15 @@ def __init__(self, *topics, **configs):
                         new_config, self.config['auto_offset_reset'])
             self.config['auto_offset_reset'] = new_config
 
+        connections_max_idle_ms = self.config['connections_max_idle_ms']
         request_timeout_ms = self.config['request_timeout_ms']
         fetch_max_wait_ms = self.config['fetch_max_wait_ms']
-        if request_timeout_ms <= fetch_max_wait_ms:
-            raise KafkaConfigurationError("Request timeout (%s) must be larger than fetch-max-wait-ms (%s)" %
-                                          (request_timeout_ms, fetch_max_wait_ms))
+        if connections_max_idle_ms <= request_timeout_ms <= fetch_max_wait_ms:
```

Comment:
This is a logical AND, meaning that it will only raise the config error if connections_max_idle_ms is <= request_timeout_ms AND request_timeout_ms <= fetch_max_wait_ms. Perhaps:

```
if not (fetch_max_wait_ms < request_timeout_ms < connections_max_idle_ms):
```

Reply:
Oh of course, I was thinking it was implementing an `OR` but that makes a lot more sense